fission-web-server/library/Fission/Web/Server/Internal/Production.hs

```diff
@@ -215,6 +216,10 @@ mkSettings logger (Server.Port port) =
   defaultSettings
     |> setPort port
     |> setLogger (Web.Log.fromLogFunc logger)
+    |> setOnExceptionResponse
+      (mapResponseHeaders (("Access-Control-Allow-Origin", "*") :) 
+        . defaultOnExceptionResponse
+      )
```

Comment:
I also have a tendency for point-free, but outside of a few cases, adding the lambda makes this stuff much more legible. A named function is even better:

```haskell
-- Why isn't this in the WAI package? Seems useful
addResponseHeader :: HeaderName -> ByteString -> Response -> Response
addResponseHeader field val = mapResponseHeaders \headers -> (field, val) : headers

-- ...

|> setLogger (Web.Log.fromLogFunc logger)
|> setOnExceptionResponse \err -> addResponseHeader $ defaultOnExceptionResponse err
|> setTimeout serverTimeout
```

Reply:
oooooh I always forget that we have BlockArguments! That actually looks super clean!

Reply:
Ahh, that actually doesn't work :slightly_frowning_face: Apparently, it'll put the next `|>` into the lambda.